deploy/likedao/templates/react-app.config.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: react-app-config-{{ .Values.deploymentTag }}
+data:
+  config.js: |
+    window.appConfig = {
+      sentry: null,
+      chainInfo: {
+        bech32Config: {
+          bech32PrefixAccAddr: "{{ .Values.reactApp.bech32Config.bech32PrefixAccAddr }}",
+          bech32PrefixAccPub: "{{ .Values.reactApp.bech32Config.bech32PrefixAccPub }}",
+          bech32PrefixValAddr: "{{ .Values.reactApp.bech32Config.bech32PrefixValAddr }}",
+          bech32PrefixValPub: "{{ .Values.reactApp.bech32Config.bech32PrefixValPub }}",
+          bech32PrefixConsAddr: "{{ .Values.reactApp.bech32Config.bech32PrefixConsAddr }}",
+          bech32PrefixConsPub: "{{ .Values.reactApp.bech32Config.bech32PrefixConsPub }}",
+        },
+        gasPrice: "{{ .Values.reactApp.walletConnect.gasPrice }}",
+        chainId: "{{ .Values.reactApp.chainId }}",
+        chainRpc: "{{ .Values.reactApp.chainRpc }}",
+        currency: {
+          coinDenom: "{{ .Values.reactApp.currency.coinDenom }}",
+          coinMinimalDenom: "{{ .Values.reactApp.currency.coinMinimalDenom }}",
+          coinDecimals: {{ .Values.reactApp.currency.coinDecimals }},
+        },
+      },
+      desmosRpc: "{{ .Values.reactApp.desmosRpc }}",
+      graphql: {
+        endpoint: "{{ .Values.reactApp.graphqlEndpoint }}",
+      },
+      authEndpoint: "{{ .Values.reactApp.authEndpoint }}",
+      chainLinks: [
+        {{- range .Values.reactApp.chainLinks }}
+          {{- with .}}
+            {
+              "chainId": "{{ .chainId }}",
+              "link": "{{ .link }}"
+            }
+          {{- end}}
+        {{- end}}
+      ],
+      footerLinks: {
+        osmosis: "{{ .Values.reactApp.footerLinks.osmosis }}",
+        liquid: "{{ .Values.reactApp.footerLinks.liquid }}",
+        emeris: "{{ .Values.reactApp.footerLinks.emeris }}",
+        contactSupport: "{{ .Values.reactApp.footerLinks.contactSupport }}",
```

Comment:
I think we should let the whole footerLinks configurable. Not making sense for oither chains just fill in osmosis or liquid link?

Most of the cosmos chain token are not listed on liquid.

Reply:
i think we can keep contactSupport as is but let's make the `getToken` section an array of object with name and link field